ofborg/src/tasks/eval/nixpkgs.rs

```diff
@@ -270,6 +270,22 @@ impl<'a> NixpkgsStrategy<'a> {
                 },
             );
 
+            if changed_paths.len() > 64 {
+                info!(
+                    "pull request has {} changed files, skipping review requests",
+                    changed_paths.len()
+                );
+                let status = CommitStatus::new(
+                    self.repo.statuses(),
+                    self.job.pr.head_sha.clone(),
+                    String::from("grahamcofborg-eval-check-maintainers"),
+                    String::from("large change, skipping maintainer review"),
```

Comment:
I like the description from the `info!` above.

```suggestion
                    String::from("large change, skipping review requests"),
```

Maybe also `s/large/massive/`? ("mass rebuild", "massive change")